tools/tools.yml

```
@@ -216,7 +216,8 @@
 
     - repo: DistrictDataLabs/yellowbrick
       site: https://www.scikit-yb.org
```

@bbengfort - as @jbednar indicated, since we haven't quite pinned down what "sponsor" means, it is up to you whether you'd like to show numfocus as your sponsor, you'd just need to add this line: 

```suggestion
      sponsors: [numfocus]
      site: https://www.scikit-yb.org
```

---

@jsignell @jbednar I really appreciate your thoughtful perspective on this and your willingness to allow us to show a sponsorship status. I agree with you on why showing sponsorship status is important in library decision making, and we primarily want our affiliation with NumFOCUS known to indicate that we have a large, stable and welcoming community that follows a specific code of conduct. 

I was reluctant to claim a status that we do not have. Although YB can apply for financial support from NumFOCUS in the same way other affiliates can we are not directly managed or supported on an ongoing basis. Our other maintainers feel the same way. I was planning on declining the status when I noticed that arviz, another NumFOCUS affiliated project, has decided to include the sponsorship logo; therefore if it's ok with you - I'd like to go ahead and do the same. 

Thank you so much for putting this site together and maintaining it!